### PR TITLE
Api branch

### PR DESCRIPTION
**Pull request by anbk77**

### CODE REVIEW

**Comment by anbk77** [comment]
[[EV-01]User signup and login API's](https://trello.com/c/4WQ8BtjR/4-ev-01user-signup-and-login-apis)

**Comment by Snehithlal** [comment]
[JWT](https://trello.com/c/9hfQm7MF/1-jwt)

File: app/controllers/api/v1/api_controller.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+module Api
+  module V1
+    class ApiController < ActionController::API
+      before_action :authenticate
+
+      rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+      rescue_from ActionController::ParameterMissing, with: :parameter_missing
+      rescue_from Errors::AuthenticateError, with: :handle_unauthenticated
+      rescue_from Errors::Unauthorized, with: :handle_unauthorized
+      # rescue_from Exception, with: :handle_exception
```
Comment (anbk77) [inline app/controllers/api/v1/api_controller.rb:12]: @Snehithlal fix this, also add errors in lib which are missing

Comment (anbk77) [reply app/controllers/api/v1/api_controller.rb:12]: check jwt sevice

**Pull request merged**